doc/run/k8s/install-sqlflow.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sqlflow
+
+---
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
Do we need to distinguish single-user or multiple-user mode in writing Kubernetes YAML fie? Or, can we use one YAML for deploying everything on minikube and Google Cloud and Alibaba Cloud?

Reply:
I think we should do this distinguish.
For SQLFlow developers, he/she can deploy SQLFlow with the single-user mode on Kubernetes, the multi-users model is no use for he/she.
For an SQLFlow playground with multi-user, we should deploy SQLFlow with the multi-users mode on Kubernetes.